_includes/overview/install.html

```diff
@@ -3,17 +3,17 @@
     <h2 class="c-heading -h1">The Node.js API Framework</h2>
     <div class="x-center">
       <div class="v-block">
-        <a class="v-button lb3">LoopBack 3<br>(Active LTS)</a>
+        <a href="/" class="v-button">LoopBack 4<br>(Active)</a>
```

Comment:
Just curious: why some link uses relative path like `/` and some uses the public url like `https://loopback.io/doc/en/lb4/Getting-started.html` =p

Reply:
I don't know why, but for some it prepends the link with `/lb3` and for others it doesn't.

Reply:
@nabdelgadir Great thank you 👍